### DISCUSSION TITLE
Klick'r 3.0.4

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#512] [Announcements]
## BugFixes

* Fix a possible crash when starting the detection with an invalid detection quality
* Handle resize when animations are disabled in developper options: https://github.com/Nain57/Smart-AutoClicker/issues/480
* Fix slow start of scenario activity: https://github.com/Nain57/Smart-AutoClicker/issues/482
* Do not execute infinite clicks infinitely during a try in dumb scenarios
* Allow to exit allow tries with volume down key

## Previous version stability

Data collected on PlayStore version only, 46K Monthly Active Users. (x2 in two weeks, thank you all!)
Left is "Users without a crash", right is "Session without a crash"

![Stability png](https://github.com/user-attachments/assets/9ebe5022-b340-4074-8a0b-a3bfa758da1a)

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.4'>Klick'r 3.0.4</a>.</em>

**Comment by noobiessss** [comment]
Great work

**Comment by shakibmolla** [comment]
this not work i try. can you add custom speed and dealy. i try telegram blum mini app mining bot not work proparly very dealy

**Reply by Nain57** [reply]
Please use the bug report form to report an issue: https://github.com/Nain57/Smart-AutoClicker/issues/new?assignees=Nain57&labels=bug&projects=&template=bug_report.yml

For feature requests, please use the relevant form as well: https://github.com/Nain57/Smart-AutoClicker/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.yml

**Comment by zhrmiah** [comment]
Thanks for the update. I love this project . Unfortunately macro is not easy to execute with any kind of quick  trigger. aIs it possible to integrate this with Tasker? Also if possible adding one click floating shortcut trigger button for  macro can help different workflow execution faster.
